### PR TITLE
Adding Policies deletion steps and other enhancements

### PR DESCRIPTION
**Pull request by mmartin24**
Implements: https://github.com/epinio/docs/issues/222

- Adding deletion process for policies
- Highlights the use of flag `--set server.stagingServiceAccountName=epinio-staging-service-account`
- Adds a mention to install Epinio as first step in the SA case.

### CODE REVIEW

File: docs/howtos/aws_iam_integration.md [hunk 1]
```diff
@@ -154,3 +172,28 @@ helm upgrade epinio epinio/epinio \
     --set server.stagingServiceAccountName=epinio-staging-service-account \
     --wait
 ```
+
+Once finished, to remove everyhing remember to remove Epinio, delete service account, detach and delete policies:
```
Comment (thehejik) [inline docs/howtos/aws_iam_integration.md:176]: typo in `everything`

Comment (mmartin24) [reply docs/howtos/aws_iam_integration.md]: thanks. fixed in:  2282f595723662907c4cdfe934fd0b367fc57370

File: docs/howtos/aws_iam_integration.md [hunk 2]
```diff
@@ -75,6 +91,8 @@ If output is returned, then you already have an IAM OIDC provider for your clust
 
 ### Epinio Server Role
 
+Ensure Epinio has been deployed first. It will be needed later to use some annotations.
```
Comment (thehejik) [inline docs/howtos/aws_iam_integration.md:94]:
```suggestion
Ensure Epinio has been deployed first. Later you will assign a new AWS Service Account to an existing Epinio installation via `helm upgrade` command.
```

Comment (mmartin24) [reply docs/howtos/aws_iam_integration.md]: looks better actually. Done in cad00d1d23b696b44f5a966722ae9193a4ada840

**Pull request merged**